commitlint.config.ts

```diff
@@ -167,6 +173,34 @@ module.exports = {
                     ]
                 },
 
+                'footer-correct-place': ({body}: {body:any}) => {
```

Comment:
@parhamsaremi I'm not convinced about the name of the rule, let's rather name it 'footer-notes-misplacement' (and as a consequence let's rename some vars in the tests)